media-libs/raylib/raylib-1.9.7.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-multilib
+
+DESCRIPTION="A simple and easy-to-use library to learn videogames programming"
+HOMEPAGE="http://www.raylib.com/"
+
+LICENSE="ZLIB"
+IUSE="alsa examples games static-libs system-glfw wayland"
+SLOT="0"
+
+if [[ ${PV} == *9999 ]] ; then
+	EGIT_REPO_URI="https://github.com/raysan5/raylib.git"
+	inherit git-r3
+else
+	SRC_URI="https://github.com/raysan5/raylib/archive/${PV}-dev.tar.gz -> ${P}.tar.gz"
+	KEYWORDS="~amd64 ~x86"
+	S="${WORKDIR}/raylib-${PV}-dev"
+fi
+
+RDEPEND="
+	alsa? ( media-libs/alsa-lib )
+	!wayland? (
+		system-glfw? ( >=media-libs/glfw-3.2.1 )
+		virtual/opengl
+		x11-libs/libX11
+		x11-libs/libXcursor
+		x11-libs/libXi
+		x11-libs/libXinerama
+		x11-libs/libXrandr
+		x11-libs/libXxf86vm
+	)
+	wayland? (
+		system-glfw? ( >=media-libs/glfw-3.2.1[wayland] )
+		dev-libs/wayland
+		media-libs/mesa[egl,wayland]
+	)
+"
+DEPEND="${RDEPEND}"
+
+multilib_src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_INSTALL_LIBDIR=$(get_libdir)
+		-DSHARED=$(usex !static-libs)
```

Comment:
This is incorrect use of USE=static-libs. The flag is supposed to enable both shared & static, not static instead of shared. That said, unless you have a very good reason to build static libs, disable them completely.